themes/Flex/templates/base.html

```diff
@@ -107,19 +107,19 @@ <h1><a href="{{ SITEURL }}">{{ SITETITLE }}</a></h1>
               {% set PAGES_SORT_ATTRIBUTE = 'title' %}
           {%- endif %}
           {% for page in pages|sort(attribute=PAGES_SORT_ATTRIBUTE) %}
-          <li><a href="{{ SITEURL }}/{{ page.url }}{% if not DISABLE_URL_HASH %}#{{ page.slug }}{% endif %}">{{ page.title }}</a></li>
+          <li><a href="{{ SITEURL }}/{{ page.url }}{% if not DISABLE_URL_HASH %}#{{ page.slug }}{% endif %}" aria-label="{{ page.title }}">{{ page.title }}</a></li>
           {% endfor %}
 
           {% for name, link in LINKS %}
-          <li><a href="{{ link }}" target="_blank">{{ name }}</a></li>
+          <li><a href="{{ link }}" aria-label="{{ name }}" target="_blank">{{ name }}</a></li>
           {% endfor %}
         </ul>
       </nav>
       {% endif %}
 
       <ul class="social">
         {% for name, link in SOCIAL %}
-        <li><a class="sc-{{ name }}" href="{{ link }}" target="_blank"><i class="fa fa-{{ name }}"></i></a></li>
+        <li><a class="sc-{{ name }}" href="{{ link }}" aria-label="{{ name }}" target="_blank"><i class="fa fa-{{ name }}"></i></a></li>
         {% endfor %}
```

Comment:
While you're here...

Would be cool if we could add this in:
```html
        <li><a class="sc-{{ name }}" href="{{ link }}" aria-label="{{ name }}" target="_blank"><i class="fa fa-{{ name }}" aria-hidden="true"></i></a></li>
```
Since icons aren't really meant to be read aloud.
(this would be a suggestion but GitHub took the wrong line and it would eat the `{% endfor %}`)

Reply:
Boom! Super possible.